### PR TITLE
manifest: Update zephyr-sdk for TFM sample changes

### PR DESCRIPTION
**Pull request by torsteingrindvik**
* Point to zephyr revision with an update to TFM samples
* Make sure the above builds by adding a necessary config when needed

### CODE REVIEW

**Review by joerchan — APPROVED** [review]
Looks good. Note that due to the update to sdk-zephyr this is blocked until the upmerge of sdk-zephyr is finished.

**Review by anangl — COMMENTED** [review]
Please do not remove the DNM label manually. @NordicBuilder will remove it after you update the zephyr module revision to the proper SHA.

File: west.yml [hunk 1]
```diff
@@ -50,7 +50,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: 3dd7043488c5d965284d0bc13926e633d2919f65
+      revision: pull/839/head
```
Comment (anangl) [inline west.yml:53]: Update to `48467b3ba546883a564a7536cae8de227e7cfc07`.

Comment (torsteingrindvik) [reply west.yml]: Updated

**Review by joerchan — APPROVED** [review]
Checked SHA

**Review by rlubos — APPROVED** [review]

**Pull request merged**